src/stylus/product-collection/hero-header.styl

```diff
@@ -16,6 +21,8 @@
 	position relative
 	top 50%
 	transform translateY(-50%)
-	@media smallTabletsAndUp
-		font-size 8em
+	@media (width >= maxscreen)
```

Comment:
`(width >= 100px)` is another POSTCSS plugin (just syntactic sugar), you can just use normal media queries here.